java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java

```diff
@@ -213,4 +224,80 @@ public interface BitmapBaseCommands {
      */
     CompletableFuture<Long> bitpos(
             String key, long bit, long start, long end, BitmapIndexType offsetType);
+
+    /**
+     * Reads or modifies the array of bits representing the string that is held at <code>key</code>
+     * based on the specified <code>subCommands</code>.
+     *
+     * @see <a href="https://redis.io/commands/bitfield/">redis.io</a> for details.
+     * @param key The key of the string.
+     * @param subCommands The <code>GET</code> subCommands to be performed on the binary value of the
+     *     string at <code>
+     *     key</code>.<br>
+     *     <ul>
+     *       <li>{@link BitFieldGet}.
+     *       <li>{@link BitFieldSet}.
+     *       <li>{@link BitFieldIncrby}.
+     *       <li>{@link BitFieldOverflow}.
+     *     </ul>
+     *
+     * @return An <code>array</code> of results from subcommands.
+     *     <ul>
+     *       <li>{@link BitFieldGet} returns the value in {@link Offset} or {@link OffsetMultiplier}.
+     *       <li>{@link BitFieldSet} returns the old value in {@link Offset} or {@link
+     *           OffsetMultiplier}.
+     *       <li>{@link BitFieldIncrby} returns the new value in {@link Offset} or {@link
+     *           OffsetMultiplier}.
+     *       <li>{@link BitFieldOverflow} determines the behaviour of <code>SET</code> and <code>
+     *           INCRBY</code> when an overflow occurs.
+     *     </ul>
+     *
+     * @example
+     *     <pre>{@code
+     * client.set("sampleKey", "A"); // "A" has binary value 01000001
+     * BitFieldSubCommands[] subcommands = new BitFieldSubCommands[] {
+     *      new BitFieldSet(new UnsignedEncoding(2), new Offset(1), 3), // Sets the new binary value to 01100001
+     *      new BitFieldGet(new UnsignedEncoding(2), new Offset(1))
+     * };
+     * Long[] payload = client.bitfield("sampleKey", subcommands).get();
+     * assertArrayEquals(payload, new Long[] {2L, 3L});
+     * }</pre>
+     */
+    CompletableFuture<Long[]> bitfield(String key, BitFieldSubCommands[] subCommands);
+
+    /**
+     * Reads the array of bits representing the string that is held at <code>key</code> based on the
+     * specified <code>subCommands</code>.
+     *
+     * @since Redis 6.0 and above
+     * @see <a href="https://redis.io/docs/latest/commands/bitfield_ro/">redis.io</a> for details.
+     * @param key The key of the string.
+     * @param subCommands The <code>GET</code> subCommands to be performed.<br>
+     *     <ul>
+     *       <li>{@link BitFieldGet}.
+     *     </ul>
+     *     <br>
+     *     Note:<br>
```

Comment:
Maybe - see thru the doc renderer how it looks better
```suggestion
     *     Note:
```